Fix vmmexpress timing

adamfd opened the pull request:
Modified:
NC template to add a step after the reboot to ensure the network stack has come up before it tries to log in with domain account - frequent issues that the script would execute before the network was up.
SLB template to add a step after the reboot to wait - numerous failures where a box would reboot and after it came up the other hosts would fail.
VMMExpress.ps1 - added a loop to wait for the network controller service to return as we had a regular failure where the script moved too fast the the variable was null

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/microsoft/SDN?pullRequest=378) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/SDN?pullRequest=378) before we can accept your contribution.<br/><br/>:x: adamfd [sign now](https://cla.opensource.microsoft.com/microsoft/SDN?pullRequest=378)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/microsoft/SDN?pullRequest=378) it.</sub>

adamfd commented:
Added a further update to the GetCertChain function in the SLB processcertificates.ps1 file.
This was causing random powershell crashes on both 2016 and 2019.
Replaced fir a dir cert instead to achieve the same thing.
